### DISCUSSION TITLE
Object file /Users/mac/.ccache/<hash>.o not in cache

### DISCUSSION THREAD

**Discussion by KorolZhu** [ccache/ccache#423] [Support]
### Question ###

i looked the log file, i see many log like that: object file filepath not in cache, but i am sure the filepath is exist, because i can open the path and find the file,  so why ?

**Comment by jrosdahl** [comment]
"Object file X not in cache" is logged if a [stat call](http://man7.org/linux/man-pages/man2/stat.2.html) returns an error. If the ccache log said `Object file /Users/mac/.ccache/a/c/45b514d28ee49aee0a12f9b34879bf-8443934.o not in cache` and you could open `/Users/mac/.ccache/a/c/45b514d28ee49aee0a12f9b34879bf-8443934.o` then something seems very fishy.

Wild guess: Maybe the file is accessible by the root user but not by the user you used when compiling?

**Comment by jrosdahl** [comment]
@KorolZhu: Are you happy with the answer or do you wonder anything else?

**Comment by KorolZhu** [comment]
Yes,  thanks, i do not know the reason.
